### ISSUE TITLE
Accounts getting banned

### ISSUE THREAD

**Issue by yusoxd** [Smug246/Luna-Grabber#919]
### What Did You Expect To Happen?

.

### What Happened?

when i grab someone's info and after it gets sent to the webhook, the person who created the webhook and server owner's discord accounts gets banned in a few hours for the reason "malicious conduct" what should i do to avoid ban?

### Add Screenshots

 .

### Which Options Did You Select?

Discord Info

### How Did You Build The File?

Pyinstaller

### Did You Bind The File?

Yes (Execute on Startup)

### Check the following before submitting (If Applicable)

- [X] I have checked the existing [issues](https://github.com/Smug246/Luna-Grabber/issues)
- [X] I have followed the [installation](https://github.com/Smug246/Luna-Grabber?tab=readme-ov-file#installation) steps
- [X] I have attempted to resolve my problem by myself

**Comment by ImTheOneWhoWatches** [comment]
Discord is cracking down on ts

**Comment by aisudhiajdc** [comment]
Maybe, STOP??!?!?

**Comment by Lolmax962** [comment]
its not an bug

**Comment by NFS11** [comment]
since when? i always use this tool. i have never once been banned for using it. also @aisudhiajdc some people wanna test this tool ig

**Comment by Olive4u** [comment]
Discord IS cracking down. Other projects like pysilon need to upgrade to evade discords snooping

**Comment by InfernoTsugikuni** [comment]
yeah, i got my main banned this way when i was playing around with it you would need to make this send the data to telegram but im too lazy to actually make it work and smug has most likely given up on this project
